docs/rn/0.56.md

```diff
@@ -0,0 +1,46 @@
+# Release 0.56
+
+:material-calendar: 2024-07-09 · :material-list-status: [Full Changelog](https://github.com/srl-labs/containerlab/releases)
+
+## Interface aliases
+
+We know, it has been a long time coming, but thanks to a brilliant contribution done by [@vista-](https://github.com/vista-) in #2124 we now have added support for [interface aliases](../manual/topo-def-file.md#interface-naming).
+
+What are interface aliases you ask? They are a way to use interface names as you see them in the NOS CLI, rather than the default `ethX` naming scheme. It is a convenience feature, but, oh boy, how much easier it is to define links when you don't have to keep doing this mental gymnastics of mapping `eth0` to `Gi0/1` and so on.
```

Comment:
Maybe how much easier _is it_ versus _it is_?
and 'this mental gymnastics' to 'the mental gymnastics'